src/Symfony/Component/Messenger/DependencyInjection/MessengerPass.php

```diff
@@ -157,7 +157,11 @@ private function registerReceivers(ContainerBuilder $container)
         $receiverMapping = array();
         foreach ($container->findTaggedServiceIds('messenger.receiver') as $id => $tags) {
             foreach ($tags as $tag) {
-                $receiverMapping[$tag['name'] ?? $id] = new Reference($id);
+                $receiverMapping[$id] = new Reference($id);
+
+                if ($tag['name'] ?? false) {
```

Comment:
I would use `isset($tag['name'])` instead.

Reply:
it wouldn't mean the same, as falsy value should be excluded.
Could be `!empty($tag['name'])` if you prefer

Reply:
(but maybe we don't care? :) )

Reply:
Exactly, the current version is wrong to me, `isset()` is semantically better here.